packages/rum-react/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@elastic/apm-rum-react",
+  "version": "0.0.0",
+  "description": "> TODO: description",
+  "author": "Hamid <h.jahtalab@gmail.com>",
+  "homepage": "https://www.elastic.co/guide/en/apm/agent/js-base/current/index.html",
+  "license": "MIT",
+  "main": "src/index.js",
```

Comment:
How are we planning to distribute the library? We need both `main` and `module` here and `cjs` files in main field.

Reply:
Added the `build:main` script